raiden/api/python.py

```diff
@@ -147,19 +147,27 @@ def leave_token_network(self, token_address, only_receiving=True):
         return connection_manager.leave(only_receiving)
 
     def get_connection_manager_funds(self, token_address):
-        """Get the connection manager for a specific token"""
-        connection_manager = self.raiden.connection_manager_for_token(token_address)
-        if not connection_manager:
-            return None
+        """Get the sum of all connection manager's open channels deposits"""
+        try:
+            connection_manager = self.raiden.connection_manager_for_token(token_address)
+        except InvalidAmount:
```

Comment:
why would the above ever raise `InvalidAmount` ? I believe you meant to catch `InvalidAddress` :)

Reply:
Oh, yea.. sorry, auto-complete fooled me here.